$NetBSD: patch-CMakeLists.txt,v 1.2 2022/03/19 23:55:42 brook Exp $

Do not version lib and share subdirs.

Do not install specially on Darwin.

--- CMakeLists.txt.orig	2018-02-13 18:38:59.000000000 +0000
+++ CMakeLists.txt
@@ -868,7 +868,7 @@ if(MSVC OR (MINGW AND OCE_INSTALL_PRIVAT
 	endif(NOT DEFINED OCE_INSTALL_PACKAGE_LIB_DIR)
 else(MSVC)
 	if(NOT DEFINED OCE_INSTALL_PACKAGE_LIB_DIR)
-		set(OCE_INSTALL_PACKAGE_LIB_DIR ${OCE_INSTALL_LIB_DIR}/oce-${OCE_VERSION})
+		set(OCE_INSTALL_PACKAGE_LIB_DIR ${OCE_INSTALL_LIB_DIR}/oce)
 	endif(NOT DEFINED OCE_INSTALL_PACKAGE_LIB_DIR)
 endif()
 
@@ -898,7 +898,7 @@ endif(NOT DEFINED OCE_INSTALL_SCRIPT_DIR
 # Data
 if(NOT DEFINED OCE_INSTALL_DATA_DIR)
 	if(NOT MSVC)
-		set(OCE_INSTALL_DATA_DIR share/oce-${OCE_VERSION})
+		set(OCE_INSTALL_DATA_DIR share/oce)
 	else ()
 		set(OCE_INSTALL_DATA_DIR "share/oce")
 	endif(NOT MSVC)
@@ -918,13 +918,8 @@ if(NOT DEFINED OCE_INSTALL_CMAKE_DATA_DI
 	if(WIN32)
 		set(OCE_INSTALL_CMAKE_DATA_DIR cmake)
 	else(WIN32)
-		if(APPLE)
-			set(OCE_INSTALL_CMAKE_DATA_DIR
-				OCE.framework/Versions/${OCE_VERSION}/Resources)
-		else(APPLE)
-			set(OCE_INSTALL_CMAKE_DATA_DIR
-				lib${LIB_SUFFIX}/oce-${OCE_VERSION})
-		endif(APPLE)
+		set(OCE_INSTALL_CMAKE_DATA_DIR
+			lib${LIB_SUFFIX}/oce)
 	endif(WIN32)
 endif(NOT DEFINED OCE_INSTALL_CMAKE_DATA_DIR)
 
